### DISCUSSION TITLE
Filter for Map Remote feature request

### DISCUSSION THREAD

**Discussion by mi-hla** [ProxymanApp/Proxyman#2165] [Ideas]
Hi,

could be possible to add **Filter** feature for **Map Remote** the same way it already works for **Map Local**?

I use Map Remote with many endpoints, and it would help me to Filter endpoints to enable/disable them without any extensive searching.

Thank you,
Michal

**Comment by NghiaTranUIT** [comment]
You can use this Beta build: https://github.com/ProxymanApp/Proxyman/issues/2157

**Reply by mi-hla** [reply]
Thank you! It will be very useful.
